bundles/org.pitest.pitclipse.core/build.properties

```diff
@@ -1,5 +1,6 @@
 source.. = src/
 bin.includes = plugin.xml,\
                META-INF/,\
+               icons/,\
```

Comment:
Looks like the build fails because you forgot to revert this file. And to update `build.properties` in the `.ui` plugin.